docs/visualize/lens.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[lens]]
+== Lens
+
+beta[]
+
+*Lens* provides you with a simple way to create basic visualizations.
+
+With Lens, you can:
+
+* Easily make sense of your data with basic visualizations.
+* Use aggregations to plot any data set, no matter how large.
+* Focus the data you want to display with searches.
+* Add your visualizations to Dashboards and Discover.
```

Comment:
Dashboard is the name of the app.  Do you instead add to a dashboard? What's it mean to add a visualization to Discover? 

Also, how about adding something about how Lens gives you suggestions, or more info about the benefits of using Lens over a visualization like Visualize > Bar chart